llms/mlx_lm/tuner/utils.py

```diff
@@ -70,6 +70,7 @@ def linear_to_lora_layers(
         "mistral",
         "llama",
         "phi",
+        "phi3",
```

Comment:
This needs to be a special case (like `phi-msft` since it has different weights (the q,k,v projections are merged).

You can follow `phi-msft` but just use the correct name.

Reply:
Alright, on it :)

Reply:
I ask because I'm yet to look into mlx lora module in detail

Reply:
Awesome!